plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -526,44 +529,89 @@ public void testTime()
                 .execute(getQueryRunner(), bigqueryViewCreateAndInsert("test.time"));
     }
 
-    @Test
-    public void testTimestampWithTimeZone()
+    @DataProvider
+    public Object[][] timestampsWithZoneDataProvider()
+    {
+        return new Object[][] {
+                {UTC},
+                {ZoneId.of("America/Los_Angeles")},
+                {ZoneId.of("America/New_York")},
+                {ZoneId.of("Europe/London")},
```

Comment:
Why did you avoid using the same value of `TestPostgreSqlTypeMapping#sessionZonesDataProvider`?

Reply:
no particular reason ... but now i know that you want the same values.